fiftyone/utils/transformers.py

```
@@ -339,56 +350,153 @@ class FiftyOneZeroShotTransformerConfig(FiftyOneTransformerConfig):
+        num_dims = len(embeddings.shape)
+        if num_dims == 2:  # Batch x Embedding
+            return embeddings
+
+        if self.config.embeddings_aggregation == "no_agg":
+            # no aggregation, just return the last hidden state
+            return embeddings
+
+        agg_fcn = (
+            np.mean if self.config.embeddings_aggregation == "mean" else np.max
+        )
+        agg_dims = (
```

Ruff (0.8.2)

Local variable `agg_dims` is assigned to but never used

Remove assignment to unused variable `agg_dims`

(F841)